tests/PedantryTest.php

```diff
@@ -3,6 +3,7 @@
 namespace MongoDB\Tests;
 
 use MongoDB;
+use PHPUnit\Framework\Attributes\DataProvider;
```

Comment:
Unrelated to this PR, but I'm wondering what `use MongoDB;` above is used for. Looks like this dates back to https://github.com/mongodb/mongo-php-library/commit/eafad0580dc4638000c195e8c0f231964616beeb.

Reply:
This prevent duplication of class names as `use` statement and in the `SKIPPED_CLASSES` list.